examples/mqtt_esp32_enterprise/mqtt_esp32_enterprise.ino

```diff
@@ -0,0 +1,90 @@
+//ESP32 Enteprise network MQTT example - eduroam compatible
+//Author: Martin Chlebovec (martinius96)
+//Web: https://arduino.php5.sk
+
+#include <WiFi.h>
+#include "esp_wpa2.h" //wpa2 library for connections to Enterprise networks
+#include <PubSubClient.h>
+
+
+char EAP_ANONYMOUS_IDENTITY[] = "anonymous@example.com";
+char EAP_IDENTITY[] = "id@example.com";
+char EAP_PASSWORD[] = "EDUROAM_PASSWORD";
+const char* ssid = "eduroam"; // eduroam SSID
+const char* mqtt_server = "mqtt.iotindustries.sk";
+
+WiFiClient espClient;
+PubSubClient client(espClient);
+unsigned long lastMsg = 0;
+char msg[50];
+int counter = 0;
+void callback(char* topic, byte* message, unsigned int length) {
+  Serial.print("Message arrived on topic: ");
+  Serial.print(topic);
+  Serial.print(". Message: ");
+  String messageTemp;
+
+  for (int i = 0; i < length; i++) {
+    Serial.print((char)message[i]);
+    messageTemp += (char)message[i];
+  }
+  Serial.println();
+}
+
+void reconnect() {
+  // Loop until we're reconnected
+  while (!client.connected()) {
+    Serial.print("Attempting MQTT connection...");
+    // Attempt to connect
+    if (client.connect("ESP8266Client")) {
```

Comment:
unique MQTT Client ID might be required, adding part of MAC addr might make it such